Cargo.toml

```diff
@@ -19,7 +19,9 @@ percent-encoding = "2.3.0"
 serde = { version = "1.0.149", features = ["derive"] }
 serde_json = "1.0.85"
 url = { version = "2.3.1" }
+import_map = { version = "0.19.0", features = ["ext"] }
```

Comment:
Moving this down here will make it easier to use in the CLI and also provide this functionality more easily to stuff like dnt.